CONTRIBUTING.md

```diff
@@ -154,48 +131,19 @@ The purpose of these guidelines is to *make reviews easier* and make the
     - Granularity helps, but it's conceptual size that matters, not extent size.
```

Comment:
I think this section is too long; also the `activity/scope-oriented` thing is a bit too abstract.